### ISSUE TITLE
Add an option to open terminal on project folder

### ISSUE THREAD

**Issue by Rubyroidas** [kitze/JSUI#91]

**Comment by kitze** [comment]
Why? The goal of this project is to stop using the terminal completely.

**Comment by Rubyroidas** [comment]
how do I update specific dependencies? not all, but what I exactly need
also, I need (for example) tag management and I don't think this app will be next git UI

**Comment by kitze** [comment]
You can update them to the last version by hovering the dependency to the latest version. do you want to update it to the latest version? 

You're right about the git part.

**Comment by juanmaguitar** [comment]
Yep, for me it would be a much more useful tool if we could open the terminal on the project folder.
- If you just want to use JSUI as a tool to organize and locate projects folders
- If you want to quickly apply some bash commands (or git, or cli's) over the projects files

In summary, to be a more flexible tool that suits more types of needs
